test/fx/test_gradual_type.py

```diff
@@ -460,5 +460,57 @@ def forward(self, x: TensorType((2, 2, 4, 5))):
                 assert isinstance(n.type, TensorType)
                 assert torch.Size(n.type.__args__) == B.forward(torch.rand(2, 2, 4, 5)).size()
 
+    def test_type_check_conv2D_and_maxpool2d(self):
```

Comment:
Same comment here as conv2d, if we can be more exhaustive about testing `stride`, `padding`, etc that would be good